src/components/userDasboard/components/DashboardDetails.jsx

```diff
@@ -55,8 +48,7 @@ const DashboardDetails = ({ numberOfOrgs, numberOfDictionary, organizations }) =
         </a>.
       </p>
       <p className="lead">
-        You currently have {numberOfDictionary} personal {dictionary}. You can{' '}
-        <Link to="dashboard/dictionaries">view all OpenMRS public dictionaries</Link>.
+        You currently have {numberOfDictionary} personal {dictionary}.
       </p>
     </div>
   );
```

Comment:
I don't think this file should be touched or anything changed, since fixes for the dashboard UI is being handled here -->  https://github.com/openmrs/openmrs-ocl-client/pull/157